src/NLog/Config/LoggingConfiguration.cs

```diff
@@ -104,6 +104,9 @@ public LoggingConfiguration()
         /// </summary>
         public IList<LoggingRule> LoggingRules { get; private set; }
```

Comment:
I think we could replace this (well the backing field) with `BlockingCollection` when we drop NET35?

Reply:
Think it is a little weird to use a producer-consumer collection for a standard IReadOnlyList. But yes another container could be nice.